homeassistant/components/notify/xmpp.py

```diff
@@ -59,11 +65,11 @@ def __init__(self):
 
             logging.basicConfig(level=logging.ERROR)
 
-            self.use_tls = True
+            self.use_tls = use_tls
             self.use_ipv6 = False
             self.add_event_handler('failed_auth', self.check_credentials)
             self.add_event_handler('session_start', self.start)
-            self.connect()
+            self.connect(use_tls=self.use_tls, use_ssl=False)
```

Comment:
> Are we always not going to use ssl ?

If one want to use a XMPP server with plain-text communication fine because the chances are high that it is a local installation. These days every responsible Jabber service provider should have switched to TLS. Not allowing SSL now does not mean we can't allow it in the future if there is a need/reason to. Hopefully this will never happen with POODLE and DROWN around.